src/import-bg.rb

```diff
@@ -29,6 +29,7 @@
       url:url,
       posZ:posZ,
       posX:posX,
+      tag:set,
```

Comment:
maybe try following the style guide you animal
```suggestion
      tag: set,
```